### PR TITLE
CGF correlation overview

### PR DESCRIPTION
**Pull request by alibow**

### CODE REVIEW

**Comment by alibow** [comment]
Hmm, it doesn't seem to link my .png files for some reason. I am not sure why, but I will try to fix it!

**Review by aflaxman — APPROVED** [review]

File: docs/source/gbd2019_models/risk_risk_correlation/child_growth_risks/index.rst [hunk 1]
```diff
@@ -35,12 +35,72 @@ Child Growth Failure Risk Correlations
...
+.. image:: bw_laz_correlation_plot.png
+
+.. image:: bw_waz_correlation_plot.png
+
+.. image:: bw_wlz_correlation_plot.png
+
+.. note::
+
+   These correlation coefficients were estimated by Chris for previous work on the :ref:`risk correlation proposal <2017_risk_models>` for the :ref:`Balanced Energy Protein simulation <2017_concept_model_vivarium_gates_bep>`.
+
```
Comment (Ninicorn) [inline docs/source/gbd2019_models/risk_risk_correlation/child_growth_risks/index.rst:76]: Perhaps reference his full name and team for future researchers.

**Review by Ninicorn — APPROVED** [review]

File: docs/source/gbd2019_models/risk_risk_correlation/child_growth_risks/index.rst (hunk 1 above)
Comment (plinds) [reply docs/source/gbd2019_models/risk_risk_correlation/child_growth_risks/index.rst:76]: I second Nicole's comment!

File: docs/source/gbd2019_models/risk_risk_correlation/child_growth_risks/index.rst [hunk 2]
```diff
@@ -35,12 +35,72 @@ Child Growth Failure Risk Correlations
...
+   * - Weight for age (WAZ)
+     - Underweight: 2+ standard deviations below mean
+
+Notably, these metrics tend to be highly correlated, with the magnitude of correlation varying with age. The metrics are related both through common causes as well as through suspected direct causal influences in some cases. The below diagram represents a causal framework for child growth metrics at birth as they relate to child growth metrics throughout the pre- and post-natal periods, as presented by [Na-et-al-2020]_.
+
+.. image:: dag_framework.png
+
+Notably, it is critical to distinguish between the correlation between these metrics attributable to shared common causes between them from correlation between these metrics attributable to direct causal relationships. However, the influence of confounding factors in the estimation of the causal relationship between these factors must be carefully considered due to the high correlation between them and the observational nature of child antropometry studies. Additionally, child age should be considered as an important effect modifier in the associations between each of these metrics, as discussed by [Alderman-and-Headey-2018]_.
```
Comment (plinds) [inline docs/source/gbd2019_models/risk_risk_correlation/child_growth_risks/index.rst:60]: typo - anthropometry

**Pull request merged**